src/tag/container.rs

```diff
@@ -282,15 +282,15 @@ impl TagContainer {
                         continue;
                     }
                     markers::MarkerType::Cue => {
-                        if marker.start_position_millis == None {
+                        if marker.start_position == None {
```

Comment:
Hmm, why did you rename this? Isn't it better to have the unit as part of the name, so it's clear that these are not samples/frames/seconds/etc.?

Reply:
Replaced by a type-safe and more expressive newtype. Just have a look at thy type definition ;)

Reply:
Well, not "newtype" but a struct that correctly *names* the value.